src/applications/personalization/profile/msw-mocks.js

```diff
@@ -28,7 +28,7 @@ export const newPaymentAccount = {
 
 const prefix = environment.API_URL;
 
-export const updateDirectDepositSuccess = [
+export const updateDD4CNPSuccess = [
```

Comment:
Do we want to add a comment that DD4CNP means Direct Deposit for Compensation and Pension? It might be confusing with no context :)

Reply:
Probably, but I'm not sure where. Given that `DD4CNP` and `DD4EDU` are used all over the place now, where do you think we should put that? I don't think we have comments explaining what `vap` or `vapSvc` mean, do we? We probably should. But again, where should that go?

Reply:
oh, do you know about the Slack `/wtf` bot? I'll see if I can add those terms to the bot. Better than nothing for now!

Reply:
Ha, yeah I thought of the wtf bot as well, but I thought that was more for organization wide acronyms.. but I don't see a reason to not add it if this is our way forward and we make a mention of it in some documentation!